packages/new-editor/src/components/editor/variables/detail/Value.tsx

```diff
@@ -0,0 +1,50 @@
+import { Fieldset, Input, SimpleSelect } from '@axonivy/ui-components';
+import { treeNodeValueAttribute } from '../../../../types/config';
+import { asEnumMetadata, metadataTypeDaytime, metadataTypeEnum, type Variable, type VariableUpdates } from '../../data/Variable';
+
+type ValueFieldsetProps = {
+  variable: Variable;
+  onChange: (updates: VariableUpdates) => void;
+};
+
+export const Value = ({ variable, onChange }: ValueFieldsetProps) => {
+  const filterEnumValuesForSelect = () => {
+    const metadata = asEnumMetadata(variable.metadata);
+    if (!metadata) {
+      return [];
+    }
+    const values = new Set(metadata.values);
+    return Array.from(values)
```

Comment:
? create a set from an array to create again an array from it?
Do you want to filter duplications here?
If yes, wrong place for this IMO. The table should not allow to create multiple times the same value.